crates/nu-parser/src/parse_patterns.rs

```diff
@@ -93,7 +93,7 @@ pub fn parse_list_pattern(working_set: &mut StateWorkingSet, span: Span) -> Matc
     let inner_span = Span::new(start, end);
     let source = working_set.get_span_contents(inner_span);
 
-    let (output, err) = lex(source, inner_span.start, &[b'\n', b'\r', b','], &[], true);
+    let (output, err) = lex(source, inner_span.start, b"\n\r,", &[], true);
```

Comment:
I am inclined to say we ignore clippy on those ones as it becomes less readable that it is kind of a set instead of a string.

Reply:
agreed. it's harder to read this way.